package.json

```diff
@@ -73,25 +78,47 @@
     "flow-bin": "^0.57.3"
   },
   "configSchema": {
+    "executablePath": {
+      "description": "Path to `flow` executable",
+      "type": "string",
+      "default": "",
+      "order": 1
+    },
     "onlyIfAppropriate": {
       "title": "Only activate when .flowconfig exists",
       "type": "boolean",
-      "default": true
+      "default": true,
+      "order": 2
     },
     "showUncovered": {
       "title": "Show uncovered code in the editor",
       "type": "boolean",
-      "default": false
-    },
-    "executablePath": {
-      "description": "Path to `flow` executable",
-      "type": "string",
-      "default": ""
+      "default": false,
+      "order": 3
     },
     "hyperclickPriority": {
       "description": "Priority to use for hyperclick provider (requires restart)",
       "type": "integer",
-      "default": "0"
+      "default": 0,
+      "order": 4
+    },
+    "outline": {
+      "title": "Outline",
+      "type": "object",
+      "order": 5,
+      "properties": {
+        "showExport": {
+          "title": "Show 'export' keyword",
+          "description": "Shows a leading 'export' keyword if the variable, function or type is exported",
+          "type": "boolean",
+          "default": false
```

Comment:
Could we enable this by default please? Looks very useful IMHO

Reply:
Kind-of ironic because I made this comment https://github.com/steelbrain/flow-ide/pull/124#pullrequestreview-76460813 but since then the way of showing exports has changed enough that it's much likeable